### DISCUSSION TITLE
Nefit Multiline is not visible on EMS BUS

### DISCUSSION THREAD

**Discussion by hendrik4711** [emsesp/EMS-ESP32#1313] [Q&A]
Hello,
I am Hendrik
My Nefit Multiline heatpump is connected on EMS.
No details show up on EMS ESP
Trendline gas heater and moduline 400 are visible.
I tried the servicekey plug and the ems connections.
All units are visible but no heatpump info.

any clues?

regards,
Hendrik

**Comment by MichaelDvP** [comment]
Please go to help, download the support info and attach it here. We need to see the device-id and the telegrams from this device.

**Comment by hendrik4711** [comment]
{
  "System Info": {
    "version": "3.6.3-dev.0",
    "platform": "ESP32-S3",
    "uptime": "001+00:14:59.284",
    "uptime (seconds)": 87299,
    "free mem": 177,
    "max alloc": 167,
    "free app": 6096,
    "reset reason": "Software reset CPU / Software reset CPU"
  },
  "Network Info": {
    "network": "WiFi",
    "hostname": "ems-esp",
    "RSSI": -36,
    "IPv4 address": "192.168.178.171/255.255.255.0",
    "IPv4 gateway": "192.168.178.1",
    "IPv4 nameserver": "192.168.178.1",
    "static ip config": false,
    "enable IPv6": false,
    "low bandwidth": false,
    "disable sleep": false,
    "enable MDNS": true,
    "enable CORS": false,
    "AP provision mode": "disconnected",
    "AP security": "wpa2",
    "AP ssid": "ems-esp"
  },
  "NTP Info": {
    "NTP status": "disconnected",
    "enabled": false,
    "server": "time.google.com",
    "tz label": "Europe/Amsterdam"
  },
  "OTA Info": {
    "enabled": true,
    "port": 8266
  },
  "MQTT Info": {
    "MQTT status": "disconnected",
    "enabled": false,
    "client id": "ems-esp",
    "keep alive": 60,
    "clean session": false,
    "entity format": 1,
    "base": "ems-esp",
    "discovery prefix": "homeassistant",
    "discovery type": 0,
    "nested format": 1,
    "ha enabled": false,
    "mqtt qos": 0,
    "mqtt retain": false,
    "publish time heartbeat": 60,
    "publish time boiler": 10,
    "publish time thermostat": 10,
    "publish time solar": 10,
    "publish time mixer": 10,
    "publish time other": 10,
    "publish time sensor": 10,
    "publish single": false,
    "publish2command": false,
    "send response": false
  },
  "Syslog Info": {
    "enabled": false
  },
  "Sensor Info": {
    "temperature sensors": 0,
    "temperature sensor reads": 0,
    "temperature sensor fails": 0,
    "analog sensors": 0,
    "analog sensor reads": 0,
    "analog sensor fails": 0
  },
  "API Info": {
    "API calls": 1,
    "API fails": 0
  },
  "Bus Info": {
    "bus status": "connected",
    "bus protocol": "Buderus",
    "bus telegrams received (rx)": 105138,
    "bus reads (tx)": 20380,
    "bus writes (tx)": 0,
    "bus incomplete telegrams": 0,
    "bus reads failed": 0,
    "bus writes failed": 0,
    "bus rx line quality": 100,
    "bus tx line quality": 100
  },
  "Settings": {
    "board profile": "S32S3",
    "locale": "en",
    "tx mode": 1,
    "ems bus id": 11,
    "shower timer": false,
    "shower alert": false,
    "hide led": false,
    "notoken api": false,
    "readonly mode": false,
    "fahrenheit": false,
    "dallas parasite": false,
    "bool format": 1,
    "bool dashboard": 1,
    "enum format": 1,
    "analog enabled": true,
    "telnet enabled": true,
    "max web log buffer": 100,
    "web log buffer": 100
  },
  "Devices": [
    {
      "type": "boiler",
      "name": "Hybrid Heatpump",
      "device id": "0x08",
      "product id": 168,
      "version": "01.05",
      "entities": 76,
      "handlers received": "0x10 0x11 0x15 0x1C 0x18 0x19 0x34 0x04",
      "handlers fetched": "0x14 0x16 0x33",
      "handlers pending": "0xBF 0xC2 0x1A 0x35 0x26 0x2A",
      "handlers ignored": "0x08 0xBC"
    },
    {
      "type": "thermostat",
      "name": "Moduline 400",
      "device id": "0x10",
      "product id": 78,
      "version": "03.05",
      "entities": 32,
      "handlers received": "0xA3 0x06 0xA2",
      "handlers fetched": "0x41 0xA7 0x40 0x3F 0x3A",
      "handlers pending": "0x12 0x13 0x38 0x39",
      "handlers ignored": "0x1A 0x35"
    },
    {
      "type": "controller",
      "name": "Hybrid Heatpump",
      "device id": "0x09",
      "product id": 168,
      "version": "01.05",
      "entities": 0,
      "handlers ignored": "0x29"
    },
    {
      "type": "heatsource",
      "name": "GBx72/Trendline/Cerapur/Greenstar Si",
      "device id": "0x70",
      "product id": 123,
      "version": "06.01",
      "entities": 0,
      "handlers pending": "0x06DC 0xE4"
    }
  ]
}

**Reply by MichaelDvP** [reply]
Hmm, looks ok to me. The heatpump is a hybrid type with 76 entities,the gasboiler is cascaded as heatsource with no entities,only managed by the hybrid. What entities are missing?

**Comment by hendrik4711** [comment]
When I remove the hybrid heatpump from the bus, I see the same messages. Then under the gasboiler.I would suppose the heatpump would show it’s own info too.E.g. settingsIf it’s on pump is workingIf the compressor is running.Errors.Just like it is shown on the moduline 400Regards,HendrikOp 4 okt 2023 om 11:45 heeft MichaelDvP ***@***.***> het volgende geschreven:﻿
Hmm, looks ok to me. The heatpump is a hybrid type with 76 entities,the gasboiler is cascaded as heatsource with no entities,only managed by the hybrid. What entities are missing?

**Reply by MichaelDvP** [reply]
As far as i understand the hybrid heating acts as a single boiler and the hybrid uses different heatsources not visible to the bus. We have added this in #495 and i can not see any new telegram or heatpump specific telegrams.

**Comment by hendrik4711** [comment]
Hm,Thanks a lot for the info and quick responses.Heatpump has all leds and info inside the unit’s closed metal enclosure.So nothing is visible from the outside when no moduline 400 on software >= 3.05 is connected as thermostat. (I use a Tado).I was hoping the gateway would help me with this.Helas.I guess/hope I can connect Tado parallel to the moduline.Moduline on status of in read only mode.Cheers,HendrikOp 4 okt 2023 om 19:02 heeft MichaelDvP ***@***.***> het volgende geschreven:﻿
As far as i understand the hybrid heating acts as a single boiler and the hybrid uses different heatsources not visible to the bus. We have added this in #495 and i can not see any new telegram or heatpump specific telegrams.
